src/ontology/mondo-ingest.Makefile

```diff
@@ -343,6 +343,17 @@ lexmatch/README.md: $(SCRIPTSDIR)/lexmatch-sssom-compare.py ../mappings/mondo-so
 
 extract-unmapped-matches: lexmatch/README.md
 
+#############################
+######### Analysis ##########
+#############################
+# todo: omim-deprecated-path: automate this using (a) `runoak -i omim.ttl obsoletes` here, then modify output, (b) fetch omim release and then do? (@dependsOn: Fix `runoak` err: Permission denied: '/.data')
```

Comment:
@matentzn I wasn't sure if I should fully automate this, but I tried automating this part after finishing everything else:

`runoak -i omim.ttl obsoletes`

I tried running this in the `makefile` goal, but got a Python error: `PermissionError: [Errno 13] Permission denied: '/.data`. [stacktrace.txt](https://github.com/monarch-initiative/mondo-ingest/files/10117610/stacktrace.txt)

I know that the error is coming from OAK, but I don't know if this is something that ODK is somehow supposed to handle, or if this really is an OAK problem. What do you think?

Reply:
we can merge now. it runs outside of ODK. when pystow issue is fixed, we can run in ODK